snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/internal/utils/UtilTest.java

```diff
@@ -16,6 +17,15 @@ public void testGetTimestamp() {
     public void testGetDateTimeFromTimestamp() {
         long timestamp = 1653923456266L;
         assertEquals("2022-05-30T15:10:56.266Z", Util.getDateTimeFromTimestamp(timestamp));
+
+        Locale defaultLocale = Locale.getDefault();
+
+        // set locale to one where different numerals are used (Egypt - arabic)
+        Locale.setDefault(new Locale("ar", "EG"));
+        assertEquals("2022-05-30T15:10:56.266Z", Util.getDateTimeFromTimestamp(timestamp));
```

Comment:
Small nitpick but I would prefer for this to have a separate test just for this behaviour (e.g., `testGetDateTimeFromTimestampAlwaysUsesEnglishLocale`). Always nicer to have smaller tests cases so that they are easier to read and understand when fail.